docs/pages/developer-guide.md

```diff
@@ -154,7 +169,7 @@ pods:
 
 * **hello-world-task**: In this example, the single pod definition is composed of a single task. The name of this task is "hello-world-task".
 
-* **goal**: Every task must have a goal state. There are two possible goal states: `RUNNING` and `FINISHED`. `RUNNING` indicates that a Task should always be running, so if it exits, it should be restarted. `FINISHED` indicates that if a task finishes successfully it does not need to be restarted.
+* **goal**: Every task must have a goal state. There are two possible goal states: `RUNNING` and `FINISHED`. `RUNNING` indicates that a task should always be running, so if it exits, it should be restarted. `FINISHED` indicates that if a task finishes successfully it does not need to be restarted.
```

Comment:
I think it'd make sense to capitalize the "Task" here as it's a specific reference to the Task that is a member of a Pod as opposed to the generic concept of some work to be completed

I could be wrong though. Suzanne may have a better idea of best style practice here.